src/UpdateAssemblyInfo/Program.cs

```diff
@@ -103,7 +103,7 @@ private static int Main(string[] args)
         private static string GetGitPath()
         {
             var envPath = Environment.GetEnvironmentVariable("PATH");
-            if (!string.IsNullOrWhiteSpace(envPath))
+            if (!string.IsNullOrWhiteSpace(envPath) && envPath.IndexOf("git.exe", StringComparison.Ordinal) > 0)

```

Comment:
Like I said, PATH contains *directory* paths. PATH does not contain *file* paths.